eitprocessing/datahandling/sequence.py

```diff
@@ -77,11 +77,9 @@ def concatenate(
         """Create a merge of two Sequence objects."""
         # TODO: rewrite
 
-        concat_eit = a.eit_data.concatenate(b.eit_data) if a.eit_data and b.eit_data else None
-        concat_continuous = (
-            a.continuous_data.concatenate(b.continuous_data) if a.continuous_data and b.continuous_data else None
-        )
-        concat_sparse = a.sparse_data.concatenate(b.sparse_data) if a.sparse_data and b.sparse_data else None
+        concat_eit = a.eit_data.concatenate(b.eit_data)
```

Comment:
Alternatively, we could check whether `a.eit_data is None` (not check as a boolean, because an empty DataCollection casts to False) or `b.eit_data is None` and create an empty DataCollection instead.